.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: CI
+# Trigger the workflow on push
+on:
+  push:
+    branches:
+      - master
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE: eclipse/che-workspace-loader
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout che-workspace-loader source code
+    - uses: actions/setup-node@v1
+      name: Configuring nodejs 8.x version
+      with:
+        node-version: '8.x'
+    - name: Get yarn cache directory path
```

Comment:
not sure we need all the yarn/nodejs stuff there if we build inside docker ?

Reply:
The idea is to cache npm packages. Do you think it useless?